doc/content/verification_and_validation/ver-1fc.md

```diff
@@ -0,0 +1,72 @@
+Another issue is that the column labels for ABAQUS and TMAP7 are reversed in Table 11 and Table 13, casting doubt on which results correspond to ABAQUS and which to TMAP7. In this benchmarking exercise, we therefore refer to these results as `ABAQUS or TMAP7 (1)` and `ABAQUS or TMAP7 (2)`. Since they are close, we still consider the benchmarking exercise successful.
+
+## Steady State solution
+
+The steady-state solution for this problem was compared to the analytical solution in addition to the ABAQUS prediction [!cite](ambrosek2008verification).
+To solve for the steady state solution for this problem, the heat flux is given by
+\begin{equation} \label{eq:solution_analytical_heat_flux}
+q''=\frac{T_{SA} - T_{SB}}{\frac{L_A}{k_A} + \frac{L_B}{k_B}},
+\end{equation}
+where
+
+    $T_{Si}$ is the temperature of surface $i$, left ($T_{SA}=600$ K) and right ($T_{SB}=0$ K),
+
+    $L_i$ is Length of segment $i$ ($L_A=L_B=40$ cm),
+
+    $k_i$ is thermal conductivity of segment $i$ ($k_A = 401$ W/m/K, $k_B = 80.2$ W/m/K).
+
+At steady state, the flux in and out of any section of the slab are equal.
+The temperature at the interface ($T_I$) can be found by setting the flux through A equal to the flux
+through B, which leads to:
+\begin{equation} \label{eq:solution_analytical_steady_state}
+\frac{T_{SA} - T_{I}}{\frac{L_A}{k_A}} = \frac{T_{I} - T_{SB}}{\frac{L_B}{k_B}}.
+\end{equation}
+The interface temperature at steady state is therefore equal to $T_I = 500$ K. The temperature profile
+for conduction in steady state, with constant physical properties, is linear. The temperature
+profile of A and B can therefore be found through linear interpolation.
+
+With TMAP8, the steady state solution can be obtained in different ways: It can be derived by using the [steady state solve](source/executioners/Steady.md) or by running a transient solution until steady state is reached.
+Ref. [!cite](ambrosek2008verification) indicates that the steady state solution was obtained by running the transient solution until $t=10,000$ s, which is what is reproduced with TMAP8 here.
+TMAP8 predictions were found to be identical to the analytical solution (the root mean square percentage error is equal to RMSPE = 0.01 %), as shown in
+[ver-1fc_comparison_temperature_steady_state].
+
+!media figures/ver-1fc_comparison_temperature_steady_state.png
+    style=width:60%;margin-bottom:2%
+    id=ver-1fc_comparison_temperature_steady_state
+    caption=Comparison of temperature profiles from the analytical solution and TMAP8 in composite structure at steady state ($t = 10000$ s).
```

Comment:
```suggestion
    caption=Comparison of temperature profiles from the analytical solution and TMAP8 in composite structure at steady state ($t = 10000$ s). RMSPE is the root mean square percentage error between the analytical and TMAP8 profiles.
```
I haven't been diligent about adding this either, but figures should always be self-explanatory.